test/distribution/test_mixture.py

```diff
@@ -127,9 +127,12 @@ def test_mixture(
 
     # check mean and stddev
     calc_mean = mixture.mean.asnumpy()
+    calc_std = mixture.std.asnumpy()
     sample_mean = samples_mix.asnumpy().mean(axis=0)
+    sample_std = np.sqrt(np.mean(np.square(samples_mix - sample_mean)))
```

Comment:
Probably `sample_std = samples_mix.asnumpy().std(axis=0)` will give you the same result